stories/core/SideNavigation.stories.tsx

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { SvgPlaceholder, SvgSettings } from '@itwin/itwinui-icons-react';
+import { useState } from '@storybook/addons';
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  SideNavigation,
+  SidenavButton,
+  SideNavigationProps,
+} from '../../src/core';
+
+export default {
+  component: SideNavigation,
+  subcomponents: { SidenavButton },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    mainItems: { control: { disable: true } },
+    secondaryItems: { control: { disable: true } },
+  },
+  args: { style: { height: 'calc(100vh - 24px) ' } },
+  title: 'Core/SideNavigation',
+} as Meta<SideNavigationProps>;
+
+export const Basic: Story<SideNavigationProps> = (args) => {
```

Comment:
Is it still needed when `ActiveItem` story shows everything?

Reply:
Could say the same about 70% of the stories in all components